app/client/src/features/accounts/UserMenu/UserMenu.tsx

```diff
@@ -179,10 +179,20 @@ export function UserMenu({ className, queryRef }: UserMenuProps) {
                     </Button>
                     <ResponsiveDialog open={type === 'login'} onClose={close}>
                         <DialogContent>
-                            <LoginForm onSuccess={() => { close(); router.reload(); }} />
+                            <LoginForm
+                                onSuccess={() => {
+                                    close();
+                                    router.reload();
+                                }}
+                            />
```

Comment:
What happened to the formatting here?

Reply:
I added in a data-test-id on the register button to do the sample test so that extra section updated the formatting.